app.py

```diff
@@ -447,23 +465,24 @@ async def view_submission(ack, body, logger, client):
     try:
         # formatting a message
         # todo: change to use json object
-        header_msg = f"*Slackblast*: "
+        header_msg = f"*Backblast!*: "
         title_msg = f"*" + title + "*"
 
-        date_msg = f"*DATE*: " + the_date
+        date_msg = f"*Date*: " + the_date
         ao_msg = f"*AO*: <#" + the_ao + ">"
-        q_msg = f"*Q*: <@" + the_q + ">"
         pax_msg = f"*PAX*: " + pax_formatted
         fngs_msg = f"*FNGs*: " + fngs
-        count_msg = f"*COUNT*: " + count
+        q_msg = f"*Q*: <@" + the_q + ">"
+        count_msg = f"*Total*: " + count
+        conditions_msg = f"*Conditions*: " + conditions
         moleskine_msg = moleskine
 
-        # Message the user via the app/bot name
+        # Message the channel as the user submitting
```

Comment:
@kodesmith, ooh, interesting. The comment was half correct before. With your change, it is still half correct. It should be `# Message the channel as the app/bot name`. It would be great to do it as the user, but we can't do that (yet?) - see #10.